sos/report/plugins/kdump.py

```diff
@@ -102,4 +103,25 @@ def setup(self):
             "/etc/default/kdump-tools"
         ])
 
+
+class CosKDump(KDump, CosPlugin):
+
+    option_list = [
+        ("all_dumps", "enable capture for all kernel dumps", "", False),
+        ("latest_dump", "enable capture for latest kernel crash dump",
+         "", False),
+    ]
```

Comment:
Plugin options changed in 4.3. They are now distinct `PluginOpt` objects. See [the boot plugin](https://github.com/sosreport/sos/blob/main/sos/report/plugins/boot.py#L22) for a good example.